suite-native/device/src/screens/DeviceInfoModalScreen.tsx

```diff
@@ -67,12 +98,22 @@ export const DeviceInfoModalScreen = () => {
 
     if (!deviceModel) return null;
 
+    const currentFwVersion = getFirmwareVersion(device);
```

Comment:
This value is a derivation of the redux state. Create selector for this please.

Reply:
b6eef5a8b